fuel/datasets/cifar10.py

```diff
@@ -33,23 +34,12 @@ class CIFAR10(H5PYDataset):
     filename = 'cifar10.hdf5'
 
     def __init__(self, which_set, **kwargs):
-        if which_set not in ('train', 'test'):
-            raise ValueError("CIFAR10 only has a train and test set")
-
         kwargs.setdefault('load_in_memory', True)
-        self.which_set = which_set
-
-        super(CIFAR10, self).__init__(self.data_path, which_set,
-                                      **kwargs)
+        super(CIFAR10, self).__init__(self.data_path, which_set, **kwargs)
+        self.default_transformers += (
```

Comment:
Maybe set this as a class attribute? I guess it will never be different between instances, and it might be useful at one point to do something like `apply_transformers(my_own_dataset, CIFAR10.default_transformers)` i.e. to be able to access the default transformers from a dataset without having to instantiate it.